tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -16,7 +16,7 @@ internal static class Debugger
             /// Default value is false (disabled).
             /// </summary>
             /// <seealso cref="DebuggerSettings.Enabled"/>
-            public const string Enabled = "DD_DEBUGGER_ENABLED";
+            public const string Enabled = "DD_DYNAMIC_INSTRUMENTATION_ENABLED";
```

Comment:
I'm sure this is wanted but - just in case - you say in the PR description that you are renaming `DD_DEBUGGER_*` variables but here you rename only this one.

Reply:
Correct, I meant 'DD_DEBUGGER_ENABLED'. 😅